engine/src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -13,6 +13,8 @@ object TerminalLayout {
   val Converter = new ThrowableProxyConverter
   Converter.start()
 
+  private val dateTimeFormatter = DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss,SS")
```

Comment:
One more note, then merging: _Do not use capital "Y" to format, or you're gonna have a bad time._ 

Further reading a link from the previous stack overflow to [this](http://stackoverflow.com/questions/26431882/difference-between-year-of-era-and-week-based-year#comment41518149_26431981) yielded:

```scala
scala> import java.sql._
import java.sql._

scala> import java.time._
import java.time._

scala> import java.time.format._
import java.time.format._

scala> val formatter = DateTimeFormatter.ofPattern("'proleptic:' u '= Week-of-Year:' Y")
formatter: java.time.format.DateTimeFormatter = 'proleptic:'' 'Value(Year)' ''= Week-of-Year:'' 'Localized(WeekBasedYear)

scala> for (i <- 2016 to 2010 by -1) {
     |     val localDate = LocalDate.of(i, 12, 31)
     |     println(formatter.format(localDate))
     | }
proleptic: 2016 = Week-of-Year: 2016
proleptic: 2015 = Week-of-Year: 2016
proleptic: 2014 = Week-of-Year: 2015
proleptic: 2013 = Week-of-Year: 2014
proleptic: 2012 = Week-of-Year: 2013
proleptic: 2011 = Week-of-Year: 2011
proleptic: 2010 = Week-of-Year: 2011

scala> 
```